magento2-nginx/fpm-7.0/etc/confd/templates/magento/env.php.tmpl

```diff
@@ -89,7 +89,7 @@ return array (
                 ),
         ),
     'x-frame-options' => 'SAMEORIGIN',
-    'MAGE_MODE' => 'default',
+    'MAGE_MODE' => '{{ getenv "MAGENTO_MODE" }}',
```

Comment:
MAGENTO_MODE is still used in the NGINX config, MAGE_MODE was set to MAGENTO_MODE's value to let magento pick it up in the command line. This PR was intended to replace #226 so MAGE_MODE wouldn't have existed.